src/BinSkim.Driver/BinSkim.Driver.csproj

```diff
@@ -7,7 +7,8 @@
     <TargetFramework>$(NetCoreVersion)</TargetFramework>
     <TargetLatestRuntimePatch>True</TargetLatestRuntimePatch>
     <OutputType>Exe</OutputType>
-    <RuntimeIdentifiers>win-x64;linux-x64;osx-x64</RuntimeIdentifiers>
+    <Platforms>x64</Platforms>
+    <SelfContained>true</SelfContained>
```

Comment:
Was this to debug the issue you saw with restore failing? The RuntimeIdentifiers need to be set and SelfContained should not need to be set always as that will slow down incremental IDE builds unnecessarily.